tests/migrations/test_state.py

```diff
@@ -22,6 +22,18 @@ class StateTests(SimpleTestCase):
     Tests state construction, rendering and modification by operations.
     """
 
+    def set_up_test_model(self, app_label, project_state):
+        project_state.add_model(ModelState(app_label, 'user', {'id': models.AutoField()}))
+        project_state.add_model(ModelState(app_label, 'comment', {
+            'id': models.AutoField(),
+            'user': models.ForeignKey('tests.user', on_delete=models.CASCADE)}))
```

Comment:
Please use hanging indentation in the new code (see https://github.com/django/django/pull/14587#discussion_r692704816).